controllers/controllers/resource/fetcher.go

```diff
@@ -162,7 +162,7 @@ func (r *CapiResourceFetcher) fetchClusterForRef(ctx context.Context, refId type
 			}
 		}
 	}
-	return nil, fmt.Errorf("eksa cluster not found for datacenterRef %v", refId)
+	return nil, fmt.Errorf("eksa cluster not found for %s: %v", kind, refId)
```

Comment:
I think `%s` would be better for `refId` as well

Reply:
Actually `refId` is a struct of type `types.NamespacedName`
That's why i left it as `%v`. It gets printed as `<namespace>/<name>`

Reply:
Interesting, in this case it behaves the same way, I didn't know this
For the record:
When a struct implements the `Stringer` interface (method `String() string`), Go will use it for both `%v` and `%s` (I tough it was only for `%s`)
Which is what is happening [in this case](https://github.com/kubernetes/apimachinery/blob/master/pkg/types/namespacedname.go#L37)